src/server/services/landingpage/qgslandingpagehandlers.cpp

```diff
@@ -69,7 +69,14 @@ json QgsLandingPageHandler::projectsData() const
   const auto constProjectKeys { availableProjects.keys() };
   for ( const auto &p : constProjectKeys )
   {
-    j.push_back( QgsLandingPageUtils::projectInfo( availableProjects[ p ] ) );
+    try
+    {
+      j.push_back( QgsLandingPageUtils::projectInfo( availableProjects[ p ] ) );
+    }
+    catch ( QgsServerException & )
+    {
+      // skip broken projects!
```

Comment:
Maybe better to log something (level: warning) unless it's already done elsewhere. We don't want broken project to go unnoticed.

Reply:
Done!